packages/viewer/src/components/scene-tree-table-cell/scene-tree-table-cell.tsx

```diff
@@ -164,7 +164,17 @@ export class SceneTreeTableCell {
               )}
             </button>
           )}
-          <div class="content">{this.displayValue()}</div>
+          {this.value && (
+            <slot>
+              <div class="content">{this.displayValue()}</div>
+            </slot>
```

Comment:
Would this be more appropriate? `<div class="content"><slot>{this.displayValue()}</slot></div>`